diff --git a/src/AuctionHouseBot.cpp b/src/AuctionHouseBot.cpp
index 4d0da17..a832f99 100644
--- a/src/AuctionHouseBot.cpp
+++ b/src/AuctionHouseBot.cpp
@@ -1825,5 +1825,6 @@ void AuctionHouseBot::Update()
         auto session = std::make_unique<WorldSession>(
             AHCharacters[botIndex].AccountID, nullptr, AccountTypes::SEC_PLAYER,
             uint8(sWorld->getIntConfig(WorldIntConfigs::CONFIG_EXPANSION)),
-            0, LOCALE_enUS, 0, false, false, false);
+            0, LOCALE_enUS, 0, false, false, false,
+            false, "", SessionKey{});
         auto player = std::make_unique<Player>(session.get());
